styles.css

```diff
@@ -0,0 +1,3 @@
+h1{
+    color: aqua;
+}
```

Comment:
- [ ] Make sure you use a class to style your heading as it is [required for the project](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).). You can add it the `h1` tag and use it to style.